www/front_src/src/Resources/Graph/Performance/Graph/annotationsAtoms.ts

```diff
@@ -0,0 +1,178 @@
+import { ScaleTime } from 'd3-scale';
+import { atom } from 'jotai';
+import {
+  always,
+  both,
+  cond,
+  dec,
+  equals,
+  find,
+  gte,
+  inc,
+  isNil,
+  last,
+  lte,
+  not,
+  or,
+  pipe,
+  Pred,
+  T,
+  __,
+} from 'ramda';
+
+import { alpha } from '@material-ui/core';
+
+import { TimelineEvent } from '../../../Details/tabs/Timeline/models';
+
+interface ChangeAnnotationHoveredProps {
+  graphWidth: number;
+  mouseX: number;
+  resourceId: string;
+  timeline: Array<TimelineEvent> | undefined;
+  xScale: ScaleTime<number, number>;
+}
+
+interface GetIsBetweenProps {
+  xEnd: number;
+  xStart: number;
+}
```

Comment:
```suggestion
interface TimeInterval {
  xEnd: number;
  xStart: number;
}
```

we can then get rid of the 'x' prefices